app/frontend/src/pages/api_data/creditsIconData.ts

```diff
@@ -0,0 +1,147 @@
+// data that will be used for credits page
+import {
+  iconArrowDown,
+  iconArrowLeft,
+  iconCheckMarkDark,
+  iconDropdownDown,
+  iconDropdownUp,
+  iconHamburgerMenu,
+  iconPlus,
+  iconSearch,
+  iconX,
+} from "assets/images/images";
+
+interface iconDatum {
+  id: number;
+  imgSrc: string;
+  name: string;
+  usedIn: string;
+  provider: string;
+  link: string;
+}
+
+const iconData: iconDatum[] = [
+  {
+    id: 1,
+    imgSrc: iconHamburgerMenu,
+    name: "Menu line",
+    usedIn: "Multiple pages",
+    provider: "Majesticons",
+    link: "https://www.majesticons.com/",
+  },
+  {
+    id: 2,
+    imgSrc: iconX,
+    name: "Cross line",
+    usedIn: "Multiple pages",
+    provider: "Majesticons",
+    link: "https://www.majesticons.com/",
+  },
+  {
+    id: 3,
+    imgSrc: iconArrowDown,
+    name: "Arrow down line",
+    usedIn: "How to Join",
+    provider: "Majesticons",
+    link: "https://www.majesticons.com/",
+  },
+  {
+    id: 4,
+    imgSrc: iconArrowLeft,
+    name: "Arrow left line",
+    usedIn: "How to Join",
+    provider: "Majesticons",
+    link: "https://www.majesticons.com/",
+  },
+  {
+    id: 5,
+    imgSrc: iconDropdownDown,
+    name: "Chevron down",
+    usedIn: "How to Join",
+    provider: "Majesticons",
+    link: "https://www.majesticons.com/",
+  },
+  {
+    id: 6,
+    imgSrc: iconDropdownUp,
+    name: "Chevron up",
+    usedIn: "How to Join",
+    provider: "Majesticons",
+    link: "https://www.majesticons.com/",
+  },
+  // // Commented out icon info shown on Figma design but not yet used in MVP
+  // {
+  //   id: 7,
+  //   imgSrc: ,
+  //   name: "Chevron left",
+  //   usedIn: "How to Join",
+  //   provider: "Majesticons",
+  //   link: "https://www.majesticons.com/",
```

Comment:
I'm okay with these staying here as comments, but any thoughts about cleaning them up as well?

Reply:
Ah, I was thinking, if I leave these commented out for now, and if and when the design team do include these icons and illustrations for future pages, it would be easier for devs to add them to the data? I can delete these if that's unnecessary.